src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -570,28 +570,103 @@ pa.scheduler.core.authentication.loginMethod=SchedulerLDAPLoginMethod
 By default, the ProActive Resource Manager stores users accounts, passwords, and group memberships (user or admin), in two files:
 
 * users and passwords accounts are stored in `PROACTIVE_HOME/config/authentication/login.cfg`.
-Each line has to follow the format *user:passwd*. The default `login.cfg` file is given hereafter:
+Each line has to follow the format *user:encypted password*.
+The default accounts in `login.cfg` file corresponds to the following passwords, which are encrypted in the actual login.cfg file:
 +
-[source]
+.login.cfg
 ----
 admin:admin
 user:pwd
 demo:demo
+guest:pwd
+test:pwd
+radmin:pwd
+nsadmin:pwd
+provider:pwd
+scheduler:scheduler_pwd
+rm:rm_pwd
+watcher:watcher_pwd
+test_executor:pwd
 ----
 
 * users membership is stored in `PROACTIVE_HOME/config/authentication/group.cfg`. For each user registered in login.cfg,
 a group membership has to be defined in this file. Each line has to look like *user:group*. Group has to be
 user to have user rights, or admin to have administrator rights. Below is an extract of `group.cfg`:
 +
-[source]
+.group.cfg
 ----
 admin:admin
+admin:nsadmins
 demo:admin
+guest:guests
+nsadmin:nsadmins
+provider:providers
+radmin:rmcoreadmins
+rm:admin
+rm:nsadmins
+scheduler:user
+test_executor:admin
 user:user
+watcher:watchers
 ----
 
 ProActive contains a set of predefined groups such as "user" and "admin". Groups are defined in `PROACTIVE_HOME/config/security.java.policy-server` as described in chapter <<User Permissions>>.
 
+In order to create new users, delete an existing user, or modify the groups or password for an existing user, a command-line tool is available.
+
+This command is available inside the *tools* folder: `PROACTIVE_HOME/tools/proactive-users`.
+
+.proactive-users
```

Comment:
The output of _proactive-client -h_ was removed recently from the documentation since it was hard to maintain it up-to-date. For the same reason, I think that _proactive-users -h_ output should not be in the documentation too.

Besides, why is _usage: proactive-users.bat_ displayed when the script  _proactive-users_ used?